src/components/forms/UpdateServiceForm.tsx

```diff
@@ -28,6 +28,12 @@ const UpdateServiceForm: Component<Props> = ({ service, onUpdateService }) => {
     UPDATE_SERVICE_MUTATION
   )
 
+  const [privateStatus, setPrivateStatus] = createSignal(service.private)
```

Comment:
I don't think this is really a status. Perhaps `privacySetting` or `privacyState`?

```suggestion
  const [privacySetting, setPrivacySetting] = createSignal(service.private)
```